### ISSUE TITLE
Proposal: Develop new UI in dedicated branch

### ISSUE THREAD

**Issue by zklosko** [LibreTime/libretime#1460]
I've been working on the new UI with Vue for a little bit and made progress until I didn't. I feel like I'm a little stuck with this part of the project right now, and I'd like to move my work to an official branch on the libretime repo for others to contribute to. The new branch would be called `dev-new-ui`.

I don't know if this would clog up our Issues and PRs with things not related to the main branch, but I think the benefits of group collaboration outweigh extra PRs. We could make an additional label to help sort out the dev work.

Here's screenshot of the main dashboard's layout (the inner section is by @hairmare). There's a handful of components, pages, popups, and a microsite page.

![Screen Shot 2021-12-03 at 11 22 37 PM](https://user-images.githubusercontent.com/31484611/147025445-df89bb7e-2cdf-4501-914b-0558229a3a1f.png)

For anyone who's interested in contributing, we're using Vue 2 with Vuetify, DayJS, Howler, Vue CLI, and Mirage JS and Storybook for dev work.

TL;DR: I need help 😁

**Comment by paddatrapper** [comment]
Is there a reason it can't go into master? Also is there a way of linking to the new UI for some views and back to the old for others? That way things could be implemented piecemeal without having to re-implement everything before the UI is ready

**Comment by zklosko** [comment]
I guess it could go directly into master. I was thinking it would be cleaner to keep it separately.

I can't think of an easy way to link both UIs, especially since the new UI is being bundled (compared to our PHP MVC) and is largely based on the new API, which still seems to be a work in progress. We're also still missing API paths for schedule info for the widgets.

**Comment by jooola** [comment]
I created the Futur project board where can create notes/issues related the the new UI / new Django API.

**Comment by zklosko** [comment]
@jooola I don't know if I can say what I have is stable. It's definitely a work in progress.

**Comment by jooola** [comment]
> @jooola I don't know if I can say what I have is stable. It's definitely a work in progress.

Yup, I meant that according to https://github.com/LibreTime/libretime/issues/1264#issuecomment-875644960 we are supposed to keep the master branch as stable as possible. This is why we can still merge everything in the main branch as long as we don't break others parts.

**Comment by zklosko** [comment]
Closing because I just discovered this request is in conflict with our C4 development standards.
